pkg/Microsoft.Private.CoreFx.OOB/Microsoft.Private.CoreFx.OOB.pkgproj

```diff
@@ -20,7 +20,11 @@
     </MSBuild>
 
     <ItemGroup>
+      <NonNetCoreAppPackage Include="Microsoft.Diagnostics.Tracing.EventSource.Redist" />
+      <NonNetCoreAppPackage Include="Microsoft.IO.Redist" />
```

Comment:
I don't think is common that we add a new package that is just for netfx?

Just trying to anticipate how common would it be that someone adds a new package for netfx and they miss to add it to this list and we break consumers of this package?

Reply:
Right but keeping costs low atk, I'm not going to do that. Let's keep it in mind. That's a valid concern.